ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviders.tsx

```diff
@@ -69,8 +83,10 @@ function AuthProviders(): ReactElement {
     const { entityId } = useParams();
     const dispatch = useDispatch();
     const { analyticsTrack } = useAnalytics();
+    const { version } = useMetadata();
 
     const [isCreateMenuOpen, setIsCreateMenuOpen] = useState(false);
+    const [isInfoExpanded, setIsInfoExpanded] = useState(false);
```

Comment:
For only one additional link (especially seeing the inconsistent spacing when expanded) what do you think about simplifying to omit the expandable section?

A change to unconditional rendering would borrow `<Flex direction={{ default: 'column' }}>…</Flex>` idiom from within the section as the child of `Alert` element to enclose:
* current horizontal flex for external and internal link
* external link to GitHub Action

If you agree, I can take a closer look at intuitive spacing which might require an optional prop.

Reply:
In the original mockups we had the expandable section since we had multiple links here. It will potentially also grow in the future, so I'd rather keep the expandable section here instead of always displaying things (to also make clear that things are somewhat "more in depth")

Reply:
Okay, here is spacing according to PatternFly design guidelines.
https://www.patternfly.org/components/expandable-section/design-guidelines#expandable-section-in-an-inline-alert

By the way, notice `title` prop has bold font weight with or without `span` element.

```diff
                  <Alert
                      isInline
                      variant="info"
                      title="Consider using short-lived tokens for machine-to-machine communications such as CI/CD pipelines, scripts, and other automation."
                  >
+                     <Flex
+                         direction={{ default: 'column' }}
+                         spaceItems={{ default: 'spaceItemsMd' }}
+                    >
                          <Flex direction={{ default: 'row' }}>…</Flex>
                          <ExpandableSection>…</ExpandableSection>
+                    </Flex>
                  </Alert>
```

Unbalanced without spacing above as counterpart to 16px below **More resources** when expanded.
![ExpandableSection_without_spaceItems](https://github.com/stackrox/stackrox/assets/11862657/7deaec46-a216-469b-ad05-80b4646b938e)

Balanced with `spaceItems={{ default: 'spaceItemsMd' }}` prop for 16px above and below.
![ExpandableSection_with_spaceItems](https://github.com/stackrox/stackrox/assets/11862657/7cd2cacd-256f-4b29-9e1d-d6e3455c883f)